docs/articles/documentation/reference/test-api/testcontroller/closewindow.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: docs
+title: t.closeWindow Method
+permalink: /documentation/reference/test-api/testcontroller/closewindow.html
+---
+
+# t.closeWindow method
+
+Closes a browser window.
+
+```js
+t.closeWindow( [windowDescriptor] )
+```
+
+Parameter | Type | Description
+--------- | ---- | ------------
+windowDescriptor *(optional)* | Object | Object that describes the target window. If this parameter is omitted, the currently active window is selected.
```

Comment:
What is the difference between an 'active' window and a 'currently active' window?